.github/workflows/node.js.yml

```diff
@@ -21,6 +21,8 @@ jobs:
         env:
             VSCODE_TEST_VERSION: ${{ matrix.vscode-version }}
             NODE_OPTIONS: '--max-old-space-size=8192'
+            AWS_TOOLKIT_TEST_CACHE_DIR: '/tmp/.vscode-test/'
```

Comment:
We _may_ be able to drop this environment variable I think and instead just use `../../.vscode-test`

Reply:
Ah this is why it was needed in codebuild: https://github.com/aws/aws-toolkit-vscode/blob/master/buildspec/linuxE2ETests.yml#L11

Reply:
The warning still exists in github ci as well (taken from the latest commits ci):
```
WARNING: IPC handle "/Users/runner/work/aws-toolkit-vscode/aws-toolkit-vscode/packages/amazonq/.vscode-test/user-data/1.91-main.sock" is longer than 103 chars, try a shorter --user-data-dir
```
so at the very least using this env variable disables the warning and any potential repercussions from it